src/pages/CustomRichTextEditorLink.tsx

```diff
@@ -1,4 +1,5 @@
 import classNames from 'classnames';
+import Image from 'next/image';
```

Comment:
Next image should have nothing to do with the rules of react.
It's a component from nextjs that also has infrastructure improvements for hosting and scaling the images.